### PR TITLE
Reverts and reports colored circuit boards

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

This pr entirely reverts #2511 and then ports the same feature, but this time I didn't just copypaste tg files.

## Why It's Good For The Game

don't just copypaste stuff, it breaks things

## Changelog
:cl:
add: Circuit boards are colored.
fix: Every machine broken by previous pr should be working now.
/:cl:

### CODE REVIEW

File: code/game/objects/items/circuitboards/machine_circuitboards.dm [hunk 1]
```diff
@@ -1240,3 +1237,7 @@
 		/obj/item/stock_parts/manipulator = /obj/item/stock_parts/manipulator/femto,
 		/obj/item/stock_parts/micro_laser = /obj/item/stock_parts/micro_laser/quadultra,
 		/obj/item/stock_parts/scanning_module = /obj/item/stock_parts/scanning_module/triphasic)
+
+/obj/item/circuitboard/machine/clockwork
+	name = "clockwork board (Report This)"
+	icon_state = "clock_mod"
```
Comment (PowerfulBacon) [inline code/game/objects/items/circuitboards/machine_circuitboards.dm:1243]: Is this ever used?

**Review by PowerfulBacon — APPROVED** [review]
Looked through it and it looks good, all the shuttle stuff is in and it fixes the broken shit that was copied over.

File: code/game/objects/items/circuitboards/machine_circuitboards.dm (hunk 1 above)
Comment (PowerfulBacon) [reply code/game/objects/items/circuitboards/machine_circuitboards.dm:1243]: actually who cares

Comment (Archanial) [reply code/game/objects/items/circuitboards/machine_circuitboards.dm:1243]: I don't know, you made clockcult.
It was here before, I didn't add anything or delete anything.

**Comment by PerishedFraud** [comment]
awesome.

**Comment by r1ks-iwnl** [comment]
What i did with my pr was pretty much copypasting the files, compiling once, fixing all the errors and when i got it to compile without a single error i thought it was good, seemed to be working in-game too so i just made the pull request, didn't think it broke shit, even asked on discord once if it works fine and i didn't get told about this. Anyway is there a better way of actually porting stuff without copypasting and duct taping code back together.

**Pull request merged**